### PR TITLE
patch: update package.json scripts

### PR DESCRIPTION
**Pull request by ctjlewis**
Currently working off of this branch - `yarn clean` reliably resolves any kind of workspace corruption by flushing caches, wiping `node_modules`, re-installing, and re-running `yarn setup`.

`yarn clear-cache` is available to just flush watchman and React/Metro caches without reinstalling `node_modules`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,8 +6,10 @@
     "setup": "yarn install && yarn allow-scripts && yarn postinstall",
     "android": "yarn uninstall:android && react-native run-android",
     "android:apk": "yarn update-tokens && yarn gradle assembleRelease && yarn uninstall:android && yarn android:load-apk",
-    "android:bundle": "./scripts/check-env.sh && yarn update-tokens && yarn gradle bundleRelease && open android/app/build/outputs/bundle/release/",
```
Comment (brunobar79) [inline package.json:9]: Why are you removing `yarn update-tokens` ? We need it.

File: package.json [hunk 2]
```diff
@@ -29,7 +31,7 @@
     "lint:ts": "yarn tsc --skipLibCheck --noEmit",
     "postinstall": "./scripts/postinstall.sh",
     "start": "react-native start",
-    "start:clean": "watchman watch-del-all && rm -rf $TMPDIR/react-* && rm -rf $TMPDIR/metro-* && rm -rf $TMPDIR/haste-map-* && react-native start -- --reset-cache",
+    "start:clean": "yarn clean && yarn start --reset-cache",
```
Comment (brunobar79) [inline package.json:34]: We def. don't want to remove `node_modules` every time we run `yarn start:clean`

**Review by brunobar79 — CHANGES_REQUESTED** [review]
Thanks for doing @ctjlewis!

With the code as it is,  I'm against merging this PR because it's very opinionated and affects our flow.  

For ex: 
- Breaks our android release script (missing update-tokens)
- I run `yarn start:clean` a lot of times a day and I don't wanna wipe node_modules every time.

I suggest you create your own aliases in your local env instead

**Comment by ctjlewis** [comment]
Sorry, let me make some changes - I copied scripts over from my local workspace, and meant to replace the update-tokens call. The `start:clean` script should also not wipe node_modules as you said.

Let me make some changes and see if it still seems superfluous. I find myself occasionally needing to flush and wipe node_modules (and so would other contributors, probably) so I would still recommend merging once it's cleaned up, and is purely additive.

File: package.json (hunk 1 above)
Comment (ctjlewis) [reply package.json:9]: It was from when I was testing the bundle script before I had env vars. I reverted this.

File: package.json (hunk 2 above)
Comment (ctjlewis) [reply package.json]: This was also reverted.

**Comment by ctjlewis** [comment]
Super sorry for that @brunobar79 - this should make more sense now, just adds `yarn clear-cache` and `yarn clean` scripts, should not conflict with anything.

**Pull request closed without merging**